core/domain/question_fetchers.py

```diff
@@ -31,29 +31,26 @@
 
 
 def get_questions_and_skill_descriptions_by_skill_ids(
-        question_count, skill_ids, start_cursor):
+        question_count, skill_ids, offset):
     """Returns the questions linked to the given skill ids.
 
     Args:
         question_count: int. The number of questions to return.
         skill_ids: list(str). The ID of the skills to which the questions are
             linked.
-        start_cursor: str. The starting point from which the batch of
-            questions are to be returned. This value should be urlsafe.
+        offset: int. Number of query results to skip.
 
     Returns:
-        list(Question), list(list(str)), str. The list of questions and the
+        list(Question), list(list(str)), int. The list of questions and the
```

Comment:
Do you really need the third arg here? (See above.)

Reply:
Removed.